paddle/fluid/inference/tests/api/CMakeLists.txt

```diff
@@ -159,55 +159,70 @@ if(WITH_MKLDNN)
   if (NOT EXISTS ${INT8_DATA_DIR})
     inference_download_and_uncompress(${INT8_DATA_DIR} "${INFERENCE_URL}/int8" "imagenet_val_100_tail.tar.gz")
   endif()
+  
+  if (NOT EXISTS ${INT8_DATA_DIR}/pascalvoc_data.bin)
+    inference_download_and_uncompress(${INT8_DATA_DIR} "${INFERENCE_URL}/int8" "pascalvoc_val_200_head.tar.gz")
```

Comment:
I see `imagenet_val_100_tail` has 100 images, but `pascalvoc_val_200_head` has 200 images, could you change to `pascalvoc_val_100_head`?

Reply:
ok ~